### DISCUSSION TITLE
Hero 10 bones max shutter angle not working

### DISCUSSION THREAD

**Discussion by seccafico8** [gopro/labs#230] [General]
as the title says when I set a max shutter angle on my hero 10 bones it doesn't limit the shutter. Compared it to my hero 11 and hero 9 and its definitely not limiting max shutter.

**Comment by dnewman-gpsw** [comment]
It is working in my test (on Bones.)

Using !MEXPT=1 for 24p capture.  I put my hand of the lens to get the longest exposure.  Looking a the SHUT metadata, the maximum exposure time was 0.0206277  or about 1/48, so 180 degree shutter.  Without this setting, the shutter would have exposed longer.

**Comment by seccafico8** [comment]
I figured out the issue. It does not change the shutter angle in the preview but it does when you actually take a video. Thanks for the help!
